### DISCUSSION TITLE
Log in to CoPilot from different users

### DISCUSSION THREAD

**Discussion by matanm28** [community/community#63239] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm using the VSCode settings sync feature with my personal GitHub account.
I have an additional enterprise account that has a Copilot subscription.
Whenever I log in to settings sync, the GitHub Copilot extension tries to connect using the GitHub token provided to settings sync instead of my enterprise account.

Getting this errors:
[INFO] [auth] [2023-08-09T08:37:52.778Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-08-09T08:37:52.782Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as matanm28."

**Comment by gowerc** [comment]
Was there ever a solution to this I am facing the same issue ?

**Reply by wattafot** [reply]
yo dude check this out: 
[https://github.com/orgs/community/discussions/6823](https://github.com/orgs/community/discussions/6823)

**Reply by jeiea** [reply]
@wattafot 
I guess this doesn't help, because when I add the expired account again to synchronize settings, it says I can't use it because it's expired. I tried disconnecting the copilot in 'Manage trusted extensions' and it was still broken.
